src/main/webapp/js/build-message-div.js

```diff
@@ -12,6 +12,9 @@ function buildMessageDiv(message){
   timeDiv.classList.add('right-align');
   timeDiv.appendChild(document.createTextNode(new Date(message.timestamp)));
 
+  const sentimentDiv= document.createElement('div');
```

Comment:
Interesting to have a sentiment score, but not sure if it would be best to display it with the messages. Perhaps we can use it on the stats page instead? For now I think this is fine though!